test/tsconfig.json

```diff
@@ -2,7 +2,11 @@
   "extends": "@tsconfig/node18",
   "compilerOptions": {
     "noEmit": true,
-    "moduleResolution": "nodenext"
+    "moduleResolution": "nodenext",
```

Comment:
We can set this to `bundler` and no longer need to add `.ts` to imports.

Reply:
Added in TS 5.0

https://www.typescriptlang.org/docs/handbook/release-notes/typescript-5-0.html#--moduleresolution-bundler